src/switchx/Switchx.hx

```diff
@@ -216,15 +221,22 @@ class Switchx {
       root.delete();
   }
       
-  public function download(version:ResolvedVersion, options:{ force: Bool }):Promise<Bool>
+  public function download(version:ResolvedVersion, options:{ force: Bool }):Promise<Bool> {
+    
+    inline function downloadArchive(url, peel, into)
```

Comment:
Right! I'd like to put that change in your hands and bug you for a release while I'm at it :)